Makefile

```diff
@@ -1,50 +1,27 @@
-# Shortcut targets
-default: test
+PACKAGE_NAME=github.com/projectcalico/libcalico-go
+GO_BUILD_VER=v0.27
 
-## Build binary
-all: test
+###############################################################################
+# Download and include Makefile.common before anything else
+###############################################################################
+MAKE_BRANCH?=$(GO_BUILD_VER)
+MAKE_REPO?=https://raw.githubusercontent.com/projectcalico/go-build/$(MAKE_BRANCH)
+
+Makefile.common: Makefile.common.$(MAKE_BRANCH)
+	cp "$<" "$@"
+Makefile.common.$(MAKE_BRANCH):
+	# Clean up any files downloaded from other branches so they don't accumulate.
+	rm -f Makefile.common.*
+	wget -nv $(MAKE_REPO)/Makefile.common -O "$@"
 
-## Run the tests
-test: vendor ut fv
+include Makefile.common
+
+###############################################################################
 
-# Define some constants
-#######################
 K8S_VERSION      ?= v1.16.0
-ETCD_VERSION     ?= v3.3.7
-COREDNS_VERSION  ?= 1.5.2
-GO_BUILD_VER     ?= v0.26
-CALICO_BUILD     ?= calico/go-build:$(GO_BUILD_VER)
-PACKAGE_NAME     ?= github.com/projectcalico/libcalico-go
-LOCAL_USER_ID    ?= $(shell id -u $$USER)
 BINDIR           ?= bin
```

Comment:
Can this be defined in Makefile.common with `bin` as the default value and redefined if needed.

Reply:
That seemed a little too close to the repository specific build rules, so I chose to leave it alone.